client/modules/workspace/src/SelectModal/SelectModal.tsx

```diff
@@ -8,18 +18,14 @@ import {
   TUser,
 } from '@client/hooks';
 
-import { BaseFileListingBreadcrumb } from '@client/common-components';
-import styles from './SelectModal.module.css';
 import {
+  BaseFileListingBreadcrumb,
   FileListingTable,
```

Comment:
Outside scope of this PR, but the file listing "Select" buttons are gray in design, i.e. secondary, instead of blue
<img width="122" alt="Screenshot 2024-05-31 at 10 30 42 AM" src="https://github.com/DesignSafe-CI/portal/assets/20326896/058f694f-d9e9-471c-b32b-721fe15ee299">

Reply:
Yes, I'll do this in next PR, might require change in common component to customize the button